frontend/src/views/web/challenge/evaluation.html

```diff
@@ -29,7 +29,7 @@ <h5><a ng-if="challenge.isChallengeHost" class="pointer theme-text fs-16"
         </div>
         <div class="row">
             <div class="col s12">
-                <div ng-bind-html="challenge.page.evaluation_details" class="fs-16 w-300 evaluation-details trix-container"></div>
+                <div mathjax-bind="challenge.page.evaluation_details" class="fs-16 w-300 evaluation-details trix-container"></div>
```

Comment:
Does `mathjax-bind` entail everything that `ng-bind-html` had?

Reply:
Yes, as far as i saw using all the possible option in trix except insert image which is broken in trix and not a mathjax-bind issue